memgpt/presets.py

```diff
@@ -1,10 +1,11 @@
 from .prompts import gpt_functions
 from .prompts import gpt_system
-from .agent import AgentAsync
+from .agent import Agent, AgentAsync
 from .utils import printd
 
 
 DEFAULT = "memgpt_chat"
+SYNC_CHAT = "memgpt_chat_sync"

```

Comment:
If I'm understanding what "present" means, doesn't it make more sense to just pass in an `async=True/False` into `use_preset`? Since async/sync is orthogonal to what the OpenAI prompts are.

Reply:
This will be removed once #154 is merged in. For now, this exists so that this change will work for autogen users and not have any merge conflicts w/#154

(Once #154 is merged in, we'll change everything to point to SyncAgent)